gateway-ha/src/main/java/io/trino/gateway/ha/domain/request/GlobalPropertyRequest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.domain.request;
+
+import io.trino.gateway.ha.router.ResourceGroupsManager;
+
+/**
+ * GlobalProperty
+ *
+ * @author Wei Peng
+ */
+public class GlobalPropertyRequest
+{
+    private String useSchema;
```

Comment:
What is the `useSchema` field used for?

Reply:
Please refer to the concept of "useScheme" in the following link: https://github.com/trinodb/trino-gateway/blob/main/docs/resource-groups-api.md.